### PR TITLE
fix OSX default temporary directories when running the integration tests

### PR DESCRIPTION
**Pull request by shadeofblue**
* task api integration test
* node integration tests

### CODE REVIEW

File: scripts/tempdir.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import os
+
+from golem.core.common import is_osx
+
+
+def fix_osx_tmpdir() -> bool:
+    if not is_osx():
+        return False
+    tmpdir = os.environ.get('TMPDIR') or ''
+    if not (tmpdir.startswith('/private') or tmpdir.startswith('/tmp')):
```
Comment (maaktweluit) [inline scripts/tempdir.py]: i think its better to only set `tmpdir` when not specified by the user, i usually run it in `/Users/me/tmp` to have the logs in a central location

```suggestion
    if not tmpdir:
```

alternatively we can rename the function to `force_tmp_dir()` then we should also allow the other 2 default docker directories, `/Users` and `/Volumes` ( not sure if that last one is required )
In this case its nice to show a warning to the user that its TMPDIR env variable is wrong and has been overridden

Sorry for the stream of thoughts, pick your best, or ignore ;)

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

One q, will approve on reply or fix

File: scripts/tempdir.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/tempdir.py]: @maaktweluit updated

**Review by maaktweluit — APPROVED** [review]

File: scripts/tempdir.py (hunk 1 above)
Comment (maaktweluit) [reply scripts/tempdir.py]: nice!

**Pull request merged**